docs/flatfield/source/reference_files.rst

```diff
@@ -1,16 +1,19 @@
 Reference File
 ==============
 The flat-field correction step uses a FLAT reference file.
+For NIRSpec data, however, there are three flat-field reference files,
+F_FLAT (fore optics), S_FLAT (spectrograph optics), and
+D_FLAT (detector).
```

Comment:
I guess those statements now need to be updated, because we're no longer including the underscore in the ref type names (i.e. FFLAT, SFLAT, DFLAT).